base/applications/mspaint/winproc.cpp

```diff
@@ -438,9 +436,7 @@ LRESULT CMainWindow::OnInitMenuPopup(UINT nMsg, WPARAM wParam, LPARAM lParam, BO
             EnableMenuItem(menu, IDM_EDITDELETESELECTION, ENABLED_IF(trueSelection));
             EnableMenuItem(menu, IDM_EDITINVERTSELECTION, ENABLED_IF(trueSelection));
             EnableMenuItem(menu, IDM_EDITCOPYTO, ENABLED_IF(trueSelection));
-            OpenClipboard();
             EnableMenuItem(menu, IDM_EDITPASTE, ENABLED_IF(IsClipboardFormatAvailable(CF_BITMAP)));
```

Comment:
It would be (for another PR maybe) of interest to support pasting from other formats than CF_BITMAP. For example, from CF_DIB, and as well from text (pasting that one should probably also open the text editor...?)

Reply:
CF_BITMAP ends in same result. CF_UNICODETEXT is interesting.

Reply:
Pasting text is not supported in WinXP Paint at all. At least not until the user opened a text editing box explicitly beforehand.